git-config/src/file/git_config.rs

```diff
@@ -1673,6 +1704,125 @@ mod from_paths {
         assert_eq!(config.len(), 1);
     }
 
+    #[test]
+    fn multiple_include() {
+        let dir = tempdir().unwrap();
+
+        let a_path = dir.path().join("a");
+        fs::write(
+            a_path.as_path(),
+            r"
+            [core]
+              a = false
+            ",
+        )
+        .expect("Unable to write config file");
+
+        let b_path = dir.path().join("b");
+        fs::write(
+            b_path.as_path(),
+            r"
+            [diff]
+              renames = true
+            ",
+        )
+        .expect("Unable to write config file");
+
+        let c_path = dir.path().join("c");
+        fs::write(
+            c_path.as_path(),
+            format!(
+                r"
+            [core]
+              c = 12
+            [include]
+              path = {}
+              path = {}
+            ",
+                a_path.as_path().to_str().unwrap(),
+                b_path.as_path().to_str().unwrap()
+            ),
+        )
+        .expect("Unable to write config file");
+
+        let config = GitConfig::from_paths(&vec![c_path.as_path()]).unwrap();
+
+        assert_eq!(
+            config.get_raw_value("core", None, "c"),
+            Ok(Cow::<[u8]>::Borrowed(b"12"))
+        );
+
+        assert_eq!(
+            config.get_raw_value("diff", None, "renames"),
+            Ok(Cow::<[u8]>::Borrowed(b"true"))
+        );
+
+        assert_eq!(
+            config.get_raw_value("core", None, "a"),
+            Ok(Cow::<[u8]>::Borrowed(b"false"))
+        );
+    }
+
+    #[test]
+    fn nested_include() {
+        let dir = tempdir().unwrap();
+
+        let a_path = dir.path().join("a");
+        fs::write(
+            a_path.as_path(),
+            r"
+            [core]
+              a = false
+              c = 1
+            ",
+        )
+        .expect("Unable to write config file");
+
+        let b_path = dir.path().join("b");
+        fs::write(
+            b_path.as_path(),
+            format!(
+                r"
+            [core]
+              b = true
+            [include]
+              path = {}
+            ",
+                a_path.as_path().to_str().unwrap()
+            ),
+        )
+        .expect("Unable to write config file");
+
+        let c_path = dir.path().join("c");
+        fs::write(
+            c_path.as_path(),
+            format!(
+                r"
+            [core]
+              c = 12
+            [include]
+              path = {}
+            ",
+                b_path.as_path().to_str().unwrap()
```

Comment:
While absolute paths are definitely great to have, I would also use relative paths from the start to be sure that isn't missed.

Please ignore if that's going to be in a separate test, but to me it seems these can be mixed quite easily for a more realistic (but more complex) test.